xcube_stac/store.py

```diff
@@ -19,30 +19,44 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import datetime
-import itertools
-import json
-from typing import Any, Container, Dict, Iterator, List, Tuple, Union
+from typing import Any, Container, Dict, Iterator, Tuple, Union
 
-import pandas as pd
+import numpy as np
 import pystac
 import pystac_client
 import requests
-from shapely.geometry import box
+import xarray as xr
+from xcube.core.mldataset import MultiLevelDataset
 from xcube.core.store import (
     DATASET_TYPE,
+    MULTI_LEVEL_DATASET_TYPE,
     DatasetDescriptor,
+    MultiLevelDatasetDescriptor,
     DataStore,
     DataStoreError,
+    DataType,
     DataTypeLike,
 )
-from xcube.util.jsonschema import JsonObjectSchema, JsonStringSchema
+from xcube.core.store.fs.impl.fs import S3FsAccessor
+from xcube.util.jsonschema import JsonObjectSchema
 
 from .constants import (
-    DATASET_OPENER_ID,
-    MIME_TYPES,
-    STAC_OPEN_PARAMETERS,
-    STAC_SEARCH_PARAMETERS,
+    DATA_OPENER_IDS,
+    STAC_STORE_PARAMETERS,
+)
+from .href_parse import _decode_href
+from .impl import SingleImplementation, StackImplementation
```

Comment:
```suggestion
from .impl import SingleImplementation
from .impl import StackImplementation
```

Reply:
Remember: except for `typings` we import either one item per line at a time or use the wrapped tuple form with parens and then also  one item per line.

Reply:
Done